inflection/tools/dictionary-parser/src/main/java/org/unicode/wikidata/ParseWikidata.java

```diff
@@ -301,6 +287,34 @@ private static void printUsage() {
             throw new IllegalArgumentException();
         }
     }
+
+    void setIgnoreProperty(String[] grammemes, Ignorable ignorable) {
+        var ignorableSet = EnumSet.of(ignorable);
+        for (String grammeme : grammemes) {
+            if (grammeme.matches("Q\\d*")) {
+                TYPEMAP.put(grammeme, ignorableSet);
+            }
+            else {
+                for (Map.Entry<String, Set<? extends Enum<?>>> entry : TYPEMAP.entrySet()) {
+                    for (var grammemeEnum : entry.getValue()) {
+                        String name = grammemeEnum.name();
+                        if (name.equalsIgnoreCase(grammeme)) {
+                            if (entry.getValue().size() == 1) {
+                                entry.setValue(ignorableSet);
+                            }
+                            else {
+                                entry.getValue().remove(grammemeEnum);
+                                ArrayList<Enum<?>> clone = new ArrayList<>(entry.getValue());
+                                clone.add(ignorable);
+                                entry.setValue(new HashSet<>(clone));
+                            }
+                            break;
+                        }
+                    }
+                }
+            }
+        }
+    }
```

Comment:
This fixes the ignorable properties and ignorable inflections, which tend to include a lot of irrelevant information.